### PR TITLE
Disable "View PDF" button when schedule page returns no results

### PR DESCRIPTION
**Pull request by andreyzb**
Steps to test:

- [ ] Test on Lily or Rose theme (Carnation includes the template override of the openy_repeat module).
- [ ] Visit `/group-exercise-classes` page
- [ ] Find the week that is empty, e.g. any day in the week should not have any event
- [ ] `Download Weekly PDF` button should not be visible

### CODE REVIEW

**Comment by andreyzb** [comment]
@podarok Please review

File: js/repeat/src/repeat.js [hunk 1]
```diff
@@ -183,36 +185,39 @@ Vue.use(VueRouter);
...
       // We add watchers dynamically otherwise initially there will be
       // up to three requests as we are changing values while initializing
       // from GET query parameters.
-      component.$watch('date', function () {
-        component.runAjaxRequest();
-        component.resetPager();
-        $('#datepicker').datepicker("setDate", moment(component.date).format('YYYY-MM-DD'));
+
+      this.$watch('date', () => {
+        //$('#datepicker').datepicker("setDate", moment(component.date).format('YYYY-MM-DD'));
```
Comment (podarok) [inline js/repeat/src/repeat.js:194]: let's avoid commented code

File: js/repeat/src/repeat.js [hunk 2]
```diff
@@ -244,7 +249,7 @@ Vue.use(VueRouter);
           return diff > -limitDays;
         }
       }).on('changeDate', function (event) {
-        // In case if use unselect date.
+        // In case if we use unselect date.
```
Comment (podarok) [inline js/repeat/src/repeat.js:252]:
```suggestion
        // In case we use unselect date.
```

**Review by podarok — CHANGES_REQUESTED** [review]

**Comment by andreyzb** [comment]
@podarok I did not rebuild repeat.js because feedback changes did not touch any logic

**Comment by podarok** [comment]
Tagged https://github.com/ynorth-projects/openy_repeat/releases/tag/2.0.11

**Pull request merged**